src/component/Viewer/Video.js

```diff
@@ -26,32 +26,34 @@ const Artplayer = React.lazy(() =>
     )
 );
 
+const domain = document.location.protocol + "//" + window.location.host;
+
 const externalPlayers = [
     {
         name: "PotPlayer",
-        url: (source, title) => `potplayer://${source}`,
+        url: (source, title) => `potplayer://${domain}${source}`,
```

Comment:
For non-local storage policy, `source` will be a complete URL. We can add a simple check if `source` is starting with letter `/`.

Reply:
Thanks, I have read the backend code,  external video player is reuse the download link.
maybe they should add a new api for specific external link?

I'm not major of frontend, wolud you like to sovle it?